Mathlib/MeasureTheory/Measure/Lebesgue/EqHaar.lean

```diff
@@ -79,6 +83,42 @@ theorem Basis.parallelepiped_basisFun (ι : Type*) [Fintype ι] :
     · exact zero_le_one
 #align basis.parallelepiped_basis_fun Basis.parallelepiped_basisFun
 
+ /-- A parallelepiped can be expressed on the standard basis. -/
+ theorem Basis.parallelepiped_eq_map  {ι E : Type*} [Fintype ι] [NormedAddCommGroup E]
+    [NormedSpace ℝ E] (b : Basis ι ℝ E) :
+    b.parallelepiped = (TopologicalSpace.PositiveCompacts.piIcc01 ι).map b.equivFun.symm
+    b.equivFunL.symm.continuous
+      (by have := FiniteDimensional.of_fintype_basis b
+          exact LinearMap.isOpenMap_of_finiteDimensional _ b.equivFun.symm.surjective) := by
```

Comment:
```suggestion
    b.parallelepiped = (TopologicalSpace.PositiveCompacts.piIcc01 ι).map b.equivFun.symm
      b.equivFunL.symm.continuous
      (by have := FiniteDimensional.of_fintype_basis b
          exact LinearMap.isOpenMap_of_finiteDimensional _ b.equivFun.symm.surjective) := by
```
to make it clear these arguments are wrapped

Reply:
Instead of the `by` block, you should be able to combine [ContinuousLinearEquiv.toHomeomorph](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Topology/Algebra/Module/Basic.html#ContinuousLinearEquiv.toHomeomorph) with [Homeomorph.isOpenMap](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Topology/Homeomorph.html#Homeomorph.isOpenMap). Actually, if you want to do it, it would probably be nice to add `ContinuousLinearEquiv.isOpenMap` to do that directly.

Reply:
As suggested, I added `ContinuousLinearEquiv.isOpenMap` and used it directly in the statement of this result.